alifedata_phyloinformatics_convert/RosettaTree.py

```diff
@@ -0,0 +1,74 @@
+import Bio
+import dendropy
+from functools import lru_cache
+import pandas
+import typing
+
+from ._aux._alifestd_validate \
+    import alifestd_validate
+from .alife_dataframe_to_biopython_tree \
+    import alife_dataframe_to_biopython_tree
+from .alife_dataframe_to_dendropy_tree \
+    import alife_dataframe_to_dendropy_tree
+from .biopython_tree_to_alife_dataframe \
+    import biopython_tree_to_alife_dataframe
+from .dendropy_tree_to_alife_dataframe \
+    import dendropy_tree_to_alife_dataframe
+
+
+class RosettaTree:
+    """Simple class for seamless interoperability between phylogenetic libraries.
+    Enables user to translate across tree implementations without repeatedly calling
+    the individual conversion functions.
+
+    Attributes
+    ----------
+    tree:
+        Tree to convert from. Can be any of the three supported by this library.
+    """
+    def __init__(self, tree: typing.Union[
+            dendropy.Tree,
+            pandas.DataFrame,
+            Bio.Phylo.BaseTree.Tree
+        ]):
+        """Construct the library-agnostic tree.
```

Comment:
I would make this just one sentence: construct tree from alife standard dataframe, biopython tree, or ...

Reply:
Done in 9398973